### PR TITLE
Added correct spacing for action toolbars.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Cleaned up the toolbars, added single class that handle toolbar spacing.

### CODE REVIEW

File: src/SmartComponents/Platform/Platform.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Title } from '@patternfly/react-core';
-import { Main } from '@red-hat-insights/insights-frontend-components';
+import { Section } from '@red-hat-insights/insights-frontend-components';
```
Comment (lgalis) [inline src/SmartComponents/Platform/Platform.js]: 'Section' is defined but never used

Comment (lgalis) [reply src/SmartComponents/Platform/Platform.js]: I'll merge once this is fixed

**Comment by Hyperkid123** [comment]
@lgalis i was editing this via GH. Please don't hate me for the extra commits 😄

**Comment by Hyperkid123** [comment]
I've just noticed code duplication i will fix it properly tomorrow...

**Pull request merged**